man/adoc/bpftrace.adoc

```diff
@@ -2841,6 +2846,21 @@ END {
 }
 ----
 
+[#probes-signal]
+=== signal
+
+.variants
+* `signal:sigusr1`
+
+Signal probes are special built-in events provided by the bpftrace runtime.
+These are triggered when a specified signal is received. Currently, only SIGUSR1 is supported.
```

Comment:
The probe is triggered only when the **bpftrace process** receives a signal, right? Perhaps it's worth mentioning here to prevent confusion.